src/translations/locales/common_sa-SA.json

```diff
@@ -0,0 +1,91 @@
+{
+  "core": {
+    "title": "Vaxx.nz",
+    "tagline": "Avanoa o Tuipuipui ole COVID-19",
+    "subtitle": "Tagai i auala e faia ai tuipuipui i nofoaga latalata ane ia te oe.",
+    "disclaimerNotAGovWebsite": "O lenei 'upega tafailagi e lei faia ele Malo. Ile faatinoina o tuipuipui asiasi ane ile upega tafailagi lea, <0>bookmyvaccine.nz</0>",
+    "loading": "utaina",
+    "getDirections": "Vaai fa'atonuga",
+    "makeABooking": "Faamauina lou avanoa",
+    "kmAway": "{{distance}} kilomita le alu",
+    "lastUpdated": "Taimi mulimuli ga fa'afouina ai - {{updatedAt}}",
+    "cancel": "Fa'aleaogaina",
+    "website": "Upega tafa'ilagi"
+  },
+  "navigation": {
+    "about": "About",
+    "contact": "Contact",
+    "getInvolved": "Get Involved",
+    "setLocation": "Faailoa lou nofoaga",
+    "setLocationConfirmation": "Faamautinoa lou nofoaga",
+    "locationModal": {
+      "locaitonTitle": "Nofoaga",
+      "locationCTA": "Tusi i lalo lou tuatusi poo le alalafaga",
+      "useCurrentLocation": "Fa'aoga le nofoaga o loo e nofo iai nei",
+      "geolocationNotSupported": "O nofoaga e le lagolagoina e le 'upega tafailagi"
+    },
+    "vaccinationsNear": "Avanoa mo tuipuipui <0>lata ane {{location}}</0>",
+    "unknownLocation": "Nofoaga le mailoa",
+    "distanceDropdown": {
+      "2km": "Totonu ole 2km",
+      "5km": "Totonu ole 5km",
+      "10km": "Totonu ole 10km",
+      "25km": "Totonu ole 25km",
+      "50km": "Totonu ole 50km",
+      "100km": "Totonu ole 100km"
+    }
+  },
+  "walkins": {
+    "sectionHeader": "Ulufale atu poo le tala'ao atu i lau ta'avale <0>- Matala ile asō</0>",
+    "openString": "Taimi matala - {{openTimeString}}",
+    "cancellationNotice": {
+      "info": "Faamolemole e aua le faia faalua se avanoa",
```

Comment:
Minor - Just for consistency I reckon we add the glottal stops to this sentence. There a couple other sentences I can see too. What do you think?

Reply:
Yeah for sure! I agree 100% on getting the glottal stops in. Happy for you to correct those if you wanted.